images/capi/ansible/roles/nvidia/tasks/main.yml

```diff
@@ -60,4 +86,13 @@
   template:
     src: templates/gridd.conf.j2
     dest: /etc/nvidia/gridd.conf
-    mode: 0644
\ No newline at end of file
+    mode: 0644
+
+- name: Cleanup packages for interacting with s3 endpoint
+  become: true
+  ansible.builtin.apt:
+    state: absent
+    purge: true
+    pkg:
+      - python3-boto3
```

Comment:
Working nuking `python3` here as well?

Reply:
I was considering that but wasn't sure what impact it'd have. 

Let me test it and then if all is good, I'll add it.

Reply:
ok, so removing python3 was deadly. It broke a number of packages. I suspect the purging of it removed more than just Python3 - for example SSH wouldn't even start probably due to some shared dependency - I didn't dig into it. The fact it wasn't starting showed it was enough of an issue here.

That being said, to prevent any confusion I've removed the install of python3 as a due diligence step as it's already part of Ubuntu anyway. I've also restricted the apt section to Ubuntu with `when: ansible_distribution == "Ubuntu"` allowing other distros to be added as required and preventing them trying to run that step and inevitably failing.